packages/ast-spec/src/parameter/RestElement/spec.ts

```diff
@@ -8,8 +8,8 @@ import type { AssignmentPattern } from '../AssignmentPattern/spec';
 export interface RestElement extends BaseNode {
   type: AST_NODE_TYPES.RestElement;
   argument: DestructuringPattern;
-  typeAnnotation?: TSTypeAnnotation;
-  optional?: boolean;
-  value?: AssignmentPattern;
-  decorators?: Decorator[];
+  typeAnnotation: TSTypeAnnotation | undefined;
+  optional: boolean;
+  value: AssignmentPattern | undefined;
+  decorators: Decorator[];
```

Comment:
this is the only decorator usage that is always a semantic error (with or without 5.0 changes).
But probs best to just leave it in until we remove it from all the params